doc/source/conf.py

```diff
@@ -25,6 +25,16 @@
 sys.path.insert(0, str(Path("../..").resolve()))
 sys.path.insert(0, str(Path("../../src").resolve()))
 
+# Here provide the default values for the Python version range
+PYTHON_DEFAULT_MAX_VERSION = 3.13
+PYTHON_DEFAULT_MIN_VERSION = 3.11
+
+# Replace e.g., |PYTHON_DEFAULT_MAX_VERSION| with actual value in .rst files
+rst_epilog = f"""
+.. |PYTHON_DEFAULT_MAX_VERSION| replace:: {PYTHON_DEFAULT_MAX_VERSION}
```

Comment:
Allow us to replace `|PYTHON_DEFAULT_MAX_VERSION|` in .rst with the actual value.

Reply:
nicely done there!